### PR TITLE
auth: test issuer endpoint instead of token endpoint

### PR DESCRIPTION
**Pull request by spadgett**
/assign @enj @benjaminapetersen

### CODE REVIEW

File: auth/auth_openshift.go [hunk 1]
```diff
@@ -88,14 +88,14 @@ func newOpenShiftAuth(ctx context.Context, c *openShiftConfig) (oauth2.Endpoint,
 	}
 
 	// Make sure we can talk to the token endpoint.
```
Comment (enj) [inline auth/auth_openshift.go]: comment is wrong now

Comment (spadgett) [reply auth/auth_openshift.go]: thank you, let me fix

**Comment by spadgett** [comment]
updatedd

**Comment by enj** [comment]
/lgtm

HEAD on issuer with `oc` works.

**Comment by spadgett** [comment]
overview flake

/retest

**Comment by spadgett** [comment]
olm flake

/retest

**Pull request merged**